README.md

```diff
@@ -17,3 +17,13 @@ console.assert(e2.code === 7);
 
 console.assert(DOMException.INUSE_ATTRIBUTE_ERR === 10);
 ```
+
+## APIs
+
+### `DOMException`
+
+The `DOMException` interface constructor.
+
+### `webidl2jsDOMException`
+
+The [interface wrapper](https://github.com/jsdom/webidl2js#for-interfaces) generated by [webidl2js](https://github.com/jsdom/webidl2js).
```

Comment:
```suggestion
The [interface wrapper API](https://github.com/jsdom/webidl2js#for-interfaces) generated by [webidl2js](https://github.com/jsdom/webidl2js).
```

(I think of "the wrapper" as the `interface` export in particular, so this attempts to make it a bit clearer. Although I think there is just some ambiguity in our terminology either way; oh well.)